tc_hivemind_backend/ingest_qdrant.py

```
@@ -47,21 +50,38 @@ def __init__(
             if True, we're clearing the cache after ingestion
+        embed_model: BaseEmbedding | None
+            the embedding model to use for the ingestion pipeline
+            default is `CohereEmbedding`
+        embed_dim: int | None
+            the embedding dimension to use for the ingestion pipeline
+            default is `None`
         """
         self.community_id = community_id
         self.qdrant_client = QdrantSingleton.get_instance().client
 
         credentials = Credentials()
```

Ruff (0.14.0)

Local variable `credentials` is assigned to but never used

Remove assignment to unused variable `credentials`

(F841)